src/api/app/models/user.rb

```diff
@@ -32,6 +32,10 @@ class User < ApplicationRecord
   has_many :groups_users, inverse_of: :user
   has_many :roles_users, inverse_of: :user
   has_many :relationships, inverse_of: :user, dependent: :destroy
+  has_many :direct_maintained_project_relationships, -> { projects.where(role_id: Role.hashed['maintainer'].id) }, class_name: 'Relationship'
```

Comment:
Could we define a scope for the relationship mode (where role is maintainer) and then call ``` user.relationships.maintained``` ?

Seems so:
```
rb(main):013:0> User.last.relationships.joins(:role).where('roles.title="maintainer"')
=> #<ActiveRecord::AssociationRelation [#<Relationship id: 5, package_id: nil, project_id: 5, role_id: 2, user_id: 7, group_id: nil>]>
```

```
User.last.direct_maintained_project_relationships
=> #<ActiveRecord::Associations::CollectionProxy [#<Relationship id: 5, package_id: nil, project_id: 5, role_id: 2, user_id: 7, group_id: nil>]>
```